nuxeo-vision-google/src/main/resources/OSGI-INF/google-vision-provider.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.vision.google.provider" version="1.0">
+
+  <require>org.nuxeo.vision.core.service</require>
```

Comment:
No need to require a component when not overriding something; it seems it looks like a simple contribution here.